README.md

```diff
@@ -20,14 +33,23 @@ This is a place for all developers of the [Microsoft Edge WebView2](https://aka.
 1. [Search for existing open bugs](https://github.com/MicrosoftEdge/WebView2Feedback/issues?q=is%3Aissue+is%3Aopen+label%3Abug) to avoid duplicates.
     1. If you find that your bug is already reported, give it a 👍 reaction, and add a comment with additional details that may help us investigate.
     1. If the issue is not already reported, [open a new issue](https://github.com/MicrosoftEdge/WebView2Feedback/issues/new/choose).
-1. Tracked issues will be labeled with the `tracked` label. If you see this label, we are aware of the issue and tracking it on our internal backlog.
+1. Tracked issues will be labeled with the <span class="label" style="background-color: #008800"> tracked </span> label. If you see this label, we are aware of the issue and tracking it on our internal backlog.
 
 ### 💡 How to request a feature
 
 1. [Search for existing feature request](https://github.com/MicrosoftEdge/WebView2Feedback/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature+request%22) to avoid duplicates.
     1. If you find a similar feature request, give it a 👍 reaction, and provide additional context into how you would use the feature.
     2. If the feature is not already requested, [open a new issue](https://github.com/MicrosoftEdge/WebView2Feedback/issues/new/choose).
-1. Tracked issues will be labeled with the `tracked` label. If you see this label, we are aware of the issue and tracking it on our internal backlog.
+1. Tracked issues will be labeled with the <span class="label" style="background-color: #008800"> tracked </span> label. If you see this label, we are aware of the issue and tracking it on our internal backlog.
+
+
+### What do the labels on the issues mean?
+
+- <span class="label" style="background-color: #008800"> tracked </span> We have acknowledged the issue and are tracking it on our internal backlog. We will consider and investigate them in the near future.
+    - For bugs, it means we have reproduced the issue.
```

Comment:
This isn't quite right - we may not be able to reproduce the issue. But it does mean, similar to feature requests, that we think there's an actual issue here.

Reply:
```suggestion
    - For bugs, it means we think this is an actual product issue.
```